### ISSUE TITLE
Some 'finally' statements have unreachable messaging

### ISSUE THREAD

**Issue by mfunk** [Esri/military-tools-geoprocessing-toolbox#204]
Several tool scripts use a *finally* statement with an unreachable message:

```
    finally:
        if debug == False and len(deleteme) > 0:
            # cleanup intermediate datasets
            if debug == True: arcpy.AddMessage("Removing intermediate datasets...")
```

Where the "AddMessage" is never reached because the 'debug' will never be both False and True.

## Expected Behavior
Pick one and stick with it.

## Current Behavior
AddMessage will never get executed in this configuration

## Possible Solution
Move it or lose it. Just add the full message with the 'if' or remove the whole line.

## Steps to Reproduce (for bugs)
N/A

## Context
More for cleanup of code than function of tool

## Your Environment
N/A

**Comment by adgiles** [comment]
Fix applied in PR #215 please verify

**Comment by NatalieCampos** [comment]
This is a code clean up task.  Looking through what was changed in the code this seems verified.  The following tools with code changes were tested and all completed fine.

1. Lowest Points
2. Add LLOS fields
3.  Add RLOS fields
4. Find Local Peaks
5. Highest Points
6.LLOS
7.RLOS

Closing issue
